Basic/Invaders/Assets/Prefabs/SceneTransitionHandler.prefab

```diff
@@ -9,8 +9,8 @@ GameObject:
   serializedVersion: 6
   m_Component:
   - component: {fileID: 7334280962834855337}
-  - component: {fileID: 7334280962834855336}
   - component: {fileID: 7334280962834855351}
+  - component: {fileID: 7334280962834855336}
```

Comment:
it seems that the order of components changed here. Is there a reason of that? Does this solve anything? Otherwise it would be better to revert this change, as changign the order of components on a gameObjects alters the execution order and can lead to bugs.

Reply:
Agree, not sure why it changed, Unity just decided to reserialize them at some point. I've reverted that change and tested the project again, which seems fine.